HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -232,9 +257,20 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
       }
     }
 
-    float HoE = 999.;
-    if (candidate.superCluster()->energy() != 0.)
-      HoE = (*HoverEMap)[candidateRef] / candidate.superCluster()->energy();
+    auto const HoE = candidate.superCluster()->energy() != 0.
+                         ? ((*HoverEMap)[candidateRef] / candidate.superCluster()->energy())
+                         : 999.;
+    if (HoE > egammaHoverECut)
+      continue;
```

Comment:
I had the idea, it was already in place. I did some checks and it seems like this is a fix to this cut. It was not done earlier.

Reply:
In the past, until `11_0_X` or so, the cut was implemented in a buggy way.. 
see here: https://github.com/cms-sw/cmssw/blob/CMSSW_11_0_X/HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc#L149-L150
it was implemented as `H < cut_on_H/E`, in the left hand side the division by supercluster energy was missing. 
So it was requested to remove the buggy cut, and it was done via this PR: https://github.com/cms-sw/cmssw/pull/31962 
But the `egammaHoverECut` parameter was not removed, that could have been removed as a cleanup of the code, since it was not being used anyway.. 
But now that the H/E cut is reintroduced properly, everything looks fine.